src/v/kafka/server/handlers/fetch.cc

```diff
@@ -66,6 +70,21 @@ make_partition_response_error(model::partition_id p_id, error_code error) {
     };
 }
 
+namespace {
+model::timeout_clock::time_point
```

Comment:
I think this method could use a comment around why we are doing this in the first place.

Reply:
added a comment